README.md

```diff
@@ -5,259 +5,77 @@
-- [Install locally with pip](#install-locally-with-pip)
-- [Install locally with poetry](#install-locally-with-poetry)
-- [Install locally with Anaconda](#install-locally-with-anaconda)
-- [Install for local development](#install-for-local-development)
-- [Deploy in a cloud instance](#deploy-in-a-cloud-instance)
-
-### Install locally with Docker
-Official Label Studio docker image is [here](https://hub.docker.com/r/heartexlabs/label-studio) and it can be downloaded with `docker pull`. 
-Run Label Studio in a Docker container and access it at `http://localhost:8080`.
-
-
-```bash
-docker pull heartexlabs/label-studio:latest
-docker run -it -p 8080:8080 -v $(pwd)/mydata:/label-studio/data heartexlabs/label-studio:latest
+For the official documentation we refer to the parent label studio [repository](https://github.com/heartexlabs/label-studio).
+
+# Deployment Guide
+The below is a guide on how to deploy the label studio instance including their ml-backends. The deployment is meant for an EC2 (or other VM) and we assume that the infrastructure has already been set up.
+
+The deployment steps are:
+1. Adjust the docker-compose.yml files for label-studio and label-studio-ml-backend to contain the right volumes.
+2. Build and run label-studio docker and label-studio-ml-backend instances
+3. Setup your label studio instance
+
+## 1. Adjusting Docker Compose
+Adjust the docker-compose files to:
+1. Mount the directory with the data to the `app` service in label studio as well as in all ml-backend services.
+The right structure is something like `/home/ubuntu/data:/label-studio/files`. `/label-studio/files` needs to have a directory called `evaluation` containing the pdfs and `test_png` containing all corresponding png files.
+2. Adjust the port for `app` on which your application shall run. If you want to publish your application and make it easily accessable your the default port 80.
+
+## 2. Build and run docker instances
+In order to build and run your docker instances on your EC2 instance, do:
+1. Clone the repositories on your EC2.
+2. Create a data directory and upload all data for your application in that repository.
+3. Create a new tmux session called ml-backend and label-studio in your EC2: `tmux new -s ml-backend` &  `tmux new -s label-studio`.
+4. Attach to the tmux session using `tmux a -t ml-backend` or `tmux a -t label-studio`. Navigate to the respective docker-compose and do `sudo docker compose build` to build all docker images.
+5. Then, inside the respective tmux session do `sudo docker compose up` and your services should be up and running.
+
+Tmux helps you keep processes alive even if you kill your shell instance. A cheatsheet regarding tmux commands can be found [here](https://tmuxcheatsheet.com).
+
+
+## 3. Setup label-studio
+1. Navigate to the URL of your EC2 (make sure you allow inbound traffic to the port on which label-studio listens in the security group of your EC2.)
+2. Create a new user, and log in using it.
```

Comment:
How to do this with the default settings that have `LABEL_STUDIO_DISABLE_SIGNUP_WITHOUT_LINK=true` ?

Reply:
I don't know. I would need to test by restarting the configuration from scratch, and we'd loose the annotation progress so far. Also I'd had to reclone the repositories. I can just add a comment that it could be that one has to change the variable first and once an initial login is created it can be changed back.